### PR TITLE
Adding some pointers for Ecplise docker tooling integration with ADB

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty lmohanty@redhat.com

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
@navidshaikh @bexelbie this should work as a stop gap solution till we have proper documentation for Eclipse docker tooling integration

File: components/centos/centos-docker-eclipse/README.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Getting started with Vagrantfile
+-----------------------------------------------------------
+
+This Vagrantfile is for integrating ADB vagrant box with `Eclipse docker tooling <https://wiki.eclipse.org/Linux_Tools_Project/Docker_Tooling>`_
```
Comment (navidshaikh) [inline components/centos/centos-docker-eclipse/README.rst]: *Vagrant

File: components/centos/centos-docker-eclipse/README.rst [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+At this moment documenation about the integration present in `ADB testing repo <https://github.com/projectatomic/adb-tests/blob/master/adb-testing/README.md>`_ .
```
Comment (navidshaikh) [inline components/centos/centos-docker-eclipse/README.rst]: *At this moment documentation about the integration is present in

File: components/centos/centos-docker-eclipse/README.rst [hunk 3]
```diff
@@ -0,0 +1,13 @@
+Getting started with Vagrantfile
+-----------------------------------------------------------
+
+This Vagrantfile is for integrating ADB vagrant box with `Eclipse docker tooling <https://wiki.eclipse.org/Linux_Tools_Project/Docker_Tooling>`_
+
+At this moment documenation about the integration present in `ADB testing repo <https://github.com/projectatomic/adb-tests/blob/master/adb-testing/README.md>`_ . 
+However we working on adding relevent documentation to ADB repo.
```
Comment (navidshaikh) [inline components/centos/centos-docker-eclipse/README.rst]: *However we are working

File: components/centos/centos-docker-eclipse/README.rst [hunk 4]
```diff
@@ -0,0 +1,13 @@
+Getting started with Vagrantfile
+-----------------------------------------------------------
+
+This Vagrantfile is for integrating ADB Vagrant box with `Eclipse docker tooling <https://wiki.eclipse.org/Linux_Tools_Project/Docker_Tooling>`_
+
+At this moment documentation about the integration is present in `ADB testing repo <https://github.com/projectatomic/adb-tests/blob/master/adb-testing/README.md>`_ .
+However we are working on adding relevent documentation to ADB repo.
```
Comment (navidshaikh) [inline components/centos/centos-docker-eclipse/README.rst]: *relevant

File: components/centos/centos-docker-eclipse/README.rst [hunk 5]
```diff
@@ -0,0 +1,13 @@
+Getting started with Vagrantfile
+-----------------------------------------------------------
+
+This Vagrantfile is for integrating ADB Vagrant box with `Eclipse docker tooling <https://wiki.eclipse.org/Linux_Tools_Project/Docker_Tooling>`_
+
```
Comment (navidshaikh) [inline components/centos/centos-docker-eclipse/README.rst:5]: add dot (.) to complete the sentence

File: components/centos/centos-docker-eclipse/README.rst [hunk 6]
```diff
@@ -0,0 +1,13 @@
+Getting started with Vagrantfile
+-----------------------------------------------------------
+
+This Vagrantfile is for integrating ADB Vagrant box with `Eclipse docker tooling <https://wiki.eclipse.org/Linux_Tools_Project/Docker_Tooling>`_.
+
+At this moment documentation about the integration is present in `ADB testing repo <https://github.com/projectatomic/adb-tests/blob/master/adb-testing/README.md>`_.
+However we are working on adding relevent documentation to ADB repo.
```
Comment (navidshaikh) [inline components/centos/centos-docker-eclipse/README.rst]: *relevant

**Comment by navidshaikh** [comment]
LGTM

**Pull request merged**